### ISSUE TITLE
Failure to open PDF created by Xerox WorkCentre 7225 scanner

### ISSUE THREAD

**Issue by toreolsensan** [hhrutter/pdfcpu#36]
Attempting to read a PDF generated by a Xerox WorkCentre 7225 scanner results in the following error:
```
Error reading the pdf: Read failed.: dereferenceObject: problem dereferencing object 6: pdfFilterPipeline: Expected DecodeParms Array corrupt
```

I've attached a pdf of a blank page.

To reproduce the error, I basically do this:
```
        cfg := pdfcpu.NewDefaultConfiguration()
        ctx, err := api.Read("Scanned.pdf", cfg)
        if err != nil {
                fmt.Printf("Error reading the pdf: %v\n", err)
        }
```

**Comment by toreolsensan** [comment]
Eh, forgot the attachment of course. :-/

[Xerox-blank-scan.pdf](https://github.com/hhrutter/pdfcpu/files/2343339/Xerox-blank-scan.pdf)

**Comment by hhrutter** [comment]
Thanks for reporting this!
It will be fixed with the upcoming release next week.

**Comment by hhrutter** [comment]
fixed by 0.1.15
